### PR TITLE
Global changes

### PR DESCRIPTION
**Pull request by eliocamp**
This fixes some global problems. 

* It removes all quotemarks around blockquotes.

* Makes sure there are no "he or she" or "him or her".

* Removes capitalization in "Quiz **A**nswers".

* Rewrites chapter overviews so that they match in style. The OO part didn't have a chapter overview so I wrote one. I don't know exactly *where* it would go, though. 

I need to solve some issues but I need your input:

* Some tables have captions (Tables 19.1 and 19.2) but the rest don't. Should every table (and figure?) have a caption?
* When citing books and other resources sometimes they are around quotes (`...I found "Concepts, Techniques and Models of Computer Programming" [@ctmcp] extremely helpful.`) while others are italicised (`...the terminology of _Extending R_ [@extending-R] and call...`). Should they all be one way or the other? Which one?

### CODE REVIEW

**Comment by juangomezduaso** [comment]
I think line 161 in the proposed  OO chapter overview has a typo ("R4" instead of "R6"), so it should be:
Chapter \@ref(r6) introduces R6, which is a radically different OO system.

**Comment by eliocamp** [comment]
Thanks!

**Comment by hadley** [comment]
I think I only added table captions when I needed to float them because they otherwise caused problems with the flow. Would you mind taking a quick look at the larger tables in the rendered PDF to see if they're causing suboptimal page breaks?

I think the copy editor used italics, so we should be consistent with them.

File: Foundations.Rmd [hunk 1]
```diff
@@ -4,37 +4,37 @@
 
 To start your journey in mastering R, the following six chapters will help you learn the foundational components of R. I expect that you've already seen many of these pieces before, but you probably have not studied them deeply. To help check your existing knowledge, each chapter starts with a quiz; if you get all the questions right, feel free to skip to the next chapter!
 
-1.  In Chapter \@ref(names-values), you'll learn about an important distinction
+1.  Chapter \@ref(names-values), teaches you about an important distinction
```
Comment (hadley) [inline Foundations.Rmd]:
```suggestion
1.  Chapter \@ref(names-values) teaches you about an important distinction
```

File: Foundations.Rmd [hunk 2]
```diff
@@ -4,37 +4,37 @@
...
+    learn how the different types of vector fit 
     together. You'll also learn about attributes, which allow you to store 
     arbitrary metadata, and form the basis for two of R's object-oriented 
     programming toolkits.
     
-1.  To write clear, concise, and efficient R code it is important to fully 
-    appreciate the power of subsetting as described in Chapter 
-    \@ref(subsetting). Understanding the fundamental components will allow you 
-    to solve new problems by combining the building blocks in novel ways.
+1.  Chapter \@ref(subsetting) describes how to use subsetting properly to write
```
Comment (hadley) [inline Foundations.Rmd]:
```suggestion
1.  Chapter \@ref(subsetting) describes how to use subsetting to write 
```

File: Foundations.Rmd [hunk 3]
```diff
@@ -4,37 +4,37 @@
...
+    components will allow you to solve new problems by combining the building 
+    blocks in novel ways.
 
-1.  The tools of control flow allow you to only execute code under certain
-    conditions, or to repeatedly execute code with changing inputs. In Chapter
-    \@ref(control-flow), you'll learn the important `if` and `for` constructs,
-    as well as related tools like `switch()` and `while`.
+1.  Chapter \@ref(control-flow) presents tools of control flow that allow you to
+    only execute code under certain conditions, or to repeatedly execute code 
+    with changing inputs: the important `if` and `for` constructs, as well as
```
Comment (hadley) [inline Foundations.Rmd]:
```suggestion
    with changing inputs. These include important `if` and `for` constructs, as well as 
```

File: Foundations.Rmd [hunk 4]
```diff
@@ -4,37 +4,37 @@
...
-    \@ref(control-flow), you'll learn the important `if` and `for` constructs,
-    as well as related tools like `switch()` and `while`.
+1.  Chapter \@ref(control-flow) presents tools of control flow that allow you to
+    only execute code under certain conditions, or to repeatedly execute code 
+    with changing inputs: the important `if` and `for` constructs, as well as 
+    related tools like `switch()` and `while`.
 
-1.  Functions are the most important building block of R code, and in Chapter
-    \@ref(functions), you'll learn exactly how they work, including the 
+1.  Chapter \@ref(functions) deals with functions -- the most important builing
```
Comment (hadley) [inline Foundations.Rmd]:
```suggestion
1.  Chapter \@ref(functions) deals with functions, the most important building 
```

File: Meta.Rmd [hunk 5]
```diff
@@ -18,28 +18,27 @@ Specifically, this book focuses on tidy evaluation (sometimes called tidy eval f
...
+1.  Chapter \@ref(meta-big-picture) gives a high level description of the whole 
+    metaprogramming story, briefly learning about all major components 
+    and how they fit together to form a cohesive whole.
+
+1.  Chapter \@ref(expressions) shows that that all R code can be described as a 
+    tree. You'll learn how to visualise these trees, how the rules of R's 
+    grammar convert linear sequences of characters into these trees, and how to 
+    use recursive functions to work with code trees.
+
+1.  Chapter \@ref(quasiquotation) presents tools from rlang you can use to
```
Comment (hadley) [inline Meta.Rmd]:
```suggestion
1.  Chapter \@ref(quasiquotation) presents tools from rlang that you can use to 
```

File: Meta.Rmd [hunk 6]
```diff
@@ -18,28 +18,27 @@ Specifically, this book focuses on tidy evaluation (sometimes called tidy eval f
...
+    tree. You'll learn how to visualise these trees, how the rules of R's 
+    grammar convert linear sequences of characters into these trees, and how to 
+    use recursive functions to work with code trees.
+
+1.  Chapter \@ref(quasiquotation) presents tools from rlang you can use to 
+    capture (quote) unevaluated function arguments. You'll also learn about
+    quasiquotation, which provides a set of techniques to unquote input to make 
+    it possible to easily generate new trees from code fragments.
+
+1.  Chapter \@ref(evaluation) moves to evaluating captured code. Here you'll
```
Comment (hadley) [inline Meta.Rmd]:
```suggestion
1.  Chapter \@ref(evaluation) moves on to evaluating captured code. Here you'll 
```

File: Meta.Rmd [hunk 7]
```diff
@@ -18,28 +18,27 @@ Specifically, this book focuses on tidy evaluation (sometimes called tidy eval f
...
+    it possible to easily generate new trees from code fragments.
+
+1.  Chapter \@ref(evaluation) moves to evaluating captured code. Here you'll 
+    learn about an important data structure, the  __quosure__, which ensures 
+    correct evaluation by capturing both the code to evaluate, and the 
+    environment in which to evaluate it. This chapter will show you how to put 
+    all the pieces together to understand how NSE works in base R, and how to 
+    write functions that work like `subset()`.
+
+1.  Chapter \@ref(translation) finally combines first-class environments, lexical
```
Comment (hadley) [inline Meta.Rmd]:
```suggestion
1.  Chapter \@ref(translation) finishes up by combining first-class environments, lexical 
```

File: OO.Rmd [hunk 8]
```diff
@@ -150,3 +150,19 @@ otype(mle_obj)
 ```
 
 Use this function to figure out which chapter to read to understand how to work with an existing object.
+
+
+1.  Chapter \@ref(base-types) first details the basic types upon which the more
```
Comment (hadley) [inline OO.Rmd]: I think this should come after the paragraph starting "The goal of this brief introductory chapter"

File: OO.Rmd [hunk 9]
```diff
@@ -150,3 +150,19 @@ otype(mle_obj)
...
+    complex objects are built. 
+    
+1.  Chapter \@ref(s3) introduces S3, which is the most simple and commonly used
```
Comment (hadley) [inline OO.Rmd]:
```suggestion
1.  Chapter \@ref(s3) introduces S3, the simplest and most commonly used
```

**Comment by eliocamp** [comment]
I'll check for big figures. You can force float by using the knitr option `fig.env = "figure"`. I can set it the default in the setup chunk, if you'd like.

**Comment by hadley** [comment]
Not figures, tables. I definitely do not want to float most figures.

File: OO.Rmd (hunk 8 above)
Comment (eliocamp) [reply OO.Rmd]: Changed!

**Comment by eliocamp** [comment]
I checked tables and italicised titles of books and stuff.

File: Expressions.Rmd [hunk 10]
```diff
@@ -380,6 +380,10 @@ The following table summarises the appearance of the different expression subtyp
 
 Both base R and rlang provide functions for testing for each type of input, although the types covered are slightly different. You can easily tell them apart because all the base functions start with `is.` and the rlang functions start with `is_`.
 
+\newpage
+
+<!-- New page so that there's no beak inside the table -->
```
Comment (hadley) [inline Expressions.Rmd]: In this case it would be better to float the table (and then reference it in the text)

File: Foundations.Rmd [hunk 11]
```diff
@@ -4,37 +4,37 @@
...
+    components will allow you to solve new problems by combining the building 
+    blocks in novel ways.
 
-1.  The tools of control flow allow you to only execute code under certain
-    conditions, or to repeatedly execute code with changing inputs. In Chapter
-    \@ref(control-flow), you'll learn the important `if` and `for` constructs,
-    as well as related tools like `switch()` and `while`.
+1.  Chapter \@ref(control-flow) presents tools of control flow that allow you to
+    only execute code under certain conditions, or to repeatedly execute code 
+    with changing inputs. These include important `if` and `for` constructs, as well as
```
Comment (hadley) [inline Foundations.Rmd]:
```suggestion
    with changing inputs. These include the important `if` and `for` constructs, as well as 
```

File: Subsetting.Rmd [hunk 12]
```diff
@@ -341,8 +341,8 @@ There are two other subsetting operators: `[[` and `$`. `[[` is used for extract
 
 `[[` is most important when working with lists because subsetting a list with `[` always returns a smaller list. To help make this easier to understand we can use a metaphor:
 
->  "If list `x` is a train carrying objects, then `x[[5]]` is
-> the object in car 5; `x[4:6]` is a train of cars 4-6." 
+>  If list `x` is a train carrying objects, then `x[[5]]` is
```
Comment (hadley) [inline Subsetting.Rmd]:
```suggestion
> If list `x` is a train carrying objects, then `x[[5]]` is
```

File: Expressions.Rmd (hunk 10 above)
Comment (eliocamp) [reply Expressions.Rmd]: I tried, but the problem is that pandoc converts markdown tables to latex using the "longtable" environment, which means that allows tables that span multiple pages and in this case it 
image

From what I've learned, this is backed into pandoc so there's no easy way to change it.

Comment (hadley) [reply Expressions.Rmd]: Ah got it, thanks for the investigation

**Pull request merged**